Production_scripts/Social_Care/Produce_social_care_lookup_tests/lookup_sc_demographics_tests.R

```diff
@@ -0,0 +1,37 @@
+####################################################
+
+# Name of file - Lookup-05-Social_Care_Demographics-tests.R
+# Original Authors - Jennifer Thom
+# Original Date - January 2022
+#
+# Written/run on - RStudio Server
+# Version of R - 3.6.1
+#
+# Description - Produce tests for social care demographic lookup file.
+
+####################################################
+
+# Packages
+library(createslf)
+library(openxlsx)
+
+
+# Read in Data---------------------------------------
+
+# Create new and old dataframes with measures for testing
+new_tests <- produce_sc_demog_lookup_tests(haven::read_sav(get_sc_demog_lookup_path()))
+old_tests <- produce_sc_demog_lookup_tests(haven::read_sav(get_sc_demog_lookup_path(update = previous_update())))
```

Comment:
This is now rds by default. This should work.
```suggestion
new_tests <- produce_sc_demog_lookup_tests(readr::read_rds(get_sc_demog_lookup_path()))
old_tests <- produce_sc_demog_lookup_tests(readr::read_rds(get_sc_demog_lookup_path(update = previous_update())))
```
Otherwise add `ext = "zsav"`